I think I am unable to successfully produce APK to test the change on my device.
```
Execution failed for task ':app:buildCMakeDebug[arm64-v8a]'.
> com.android.ide.common.process.ProcessException: ninja: Entering directory `/home/kamil/Projects/luajit-kamil/app/.cxx/Debug/1r284r2h/arm64-v8a'
  [1/1] Linking C shared library /home/kamil/Projects/luajit-kamil/app/build/intermediates/cxx/Debug/1r284r2h/obj/arm64-v8a/libluajit-launcher.so
  FAILED: /home/kamil/Projects/luajit-kamil/app/build/intermediates/cxx/Debug/1r284r2h/obj/arm64-v8a/libluajit-launcher.so 
  : && /home/kamil/Android/Sdk/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/clang --target=aarch64-none-linux-android21 --gcc-toolchain=/home/kamil/Android/Sdk/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64 --sysroot=/home/kamil/Android/Sdk/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/sysroot -fPIC -g -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security  -fno-limit-debug-info  -static-libstdc++ -Wl,--build-id=sha1 -Wl,--no-rosegment -Wl,--fatal-warnings -Wl,--no-undefined -Qunused-arguments -shared -Wl,-soname,libluajit-launcher.so -o /home/kamil/Projects/luajit-kamil/app/build/intermediates/cxx/Debug/1r284r2h/obj/arm64-v8a/libluajit-launcher.so CMakeFiles/luajit-launcher.dir/android_native_app_glue/android_native_app_glue.c.o CMakeFiles/luajit-launcher.dir/jni_helper.c.o CMakeFiles/luajit-launcher.dir/main.c.o  -landroid  -llog  -lluajit-5.1  -pthread  -latomic -lm && :
  ld: error: unable to find library -lluajit-5.1
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
  ninja: build stopped: subcommand failed.
```
The missing package seems to be installed:
```
kamil@fedora:~$ dnf repoquery --installed lua* luajit*
lua-0:5.4.8-4.fc42.x86_64
lua-devel-0:5.4.8-4.fc42.x86_64
lua-lgi-compat-0:0.9.2-22.fc42.x86_64
lua-libs-0:5.4.8-4.fc42.x86_64
lua-rpm-macros-0:1-15.fc42.noarch
lua-srpm-macros-0:1-15.fc42.noarch
luajit-0:2.1.1761727121-1.fc42.x86_64
luajit-devel-0:2.1.1761727121-1.fc42.x86_64
luajit2.1-luv-0:1.51.0.0-2.fc42.x86_64

kamil@fedora:~$ pkg-config --cflags --libs luajit
-I/usr/include/luajit-2.1 -lluajit-5.1
```
Do you see anything wrong here?
Is there any step by step instruction how to build the project?

--- a/app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt
+++ b/app/src/main/java/org/koreader/launcher/device/DeviceInfo.kt
@@ -109,6 +109,7 @@ object DeviceInfo {
         ONYX_NOTE5,
         ONYX_NOTE_AIR,
         ONYX_NOTE_AIR2,
+        ONYX_NOTE_AIR_3,
         ONYX_NOTE_AIR_3C,
         ONYX_NOTE_AIR_4C,
         ONYX_NOTE_AIR_5C,
@@ -502,6 +503,10 @@ object DeviceInfo {
             BRAND == "onyx" && (MODEL == "noteair2" || MODEL == "noteair2p")
             -> Id.ONYX_NOTE_AIR2
 
+            // Onyx Note Air 3
+            BRAND == "onyx" && MODEL == "noteair3"
+            -> Id.ONYX_NOTE_AIR_3
+
             // Onyx Note Air 3C
             BRAND == "onyx" && MODEL == "noteair3c"
             -> Id.ONYX_NOTE_AIR_3C
@@ -509,7 +514,7 @@ object DeviceInfo {
             // Onyx Note Air 4C
             BRAND == "onyx" && MODEL == "noteair4c"
             -> Id.ONYX_NOTE_AIR_4C
-            
+
             // Onyx Note Air 5C
             BRAND == "onyx" && MODEL == "noteair5c"
             -> Id.ONYX_NOTE_AIR_5C

--- a/app/src/main/java/org/koreader/launcher/device/EPDFactory.kt
+++ b/app/src/main/java/org/koreader/launcher/device/EPDFactory.kt
@@ -121,6 +121,7 @@ object EPDFactory {
                 DeviceInfo.Id.ONYX_NOTE5,
                 DeviceInfo.Id.ONYX_NOTE_AIR,
                 DeviceInfo.Id.ONYX_NOTE_AIR2,
+                DeviceInfo.Id.ONYX_NOTE_AIR_3,
                 DeviceInfo.Id.ONYX_NOTE_AIR_3C,
                 DeviceInfo.Id.ONYX_NOTE_AIR_4C,
                 DeviceInfo.Id.ONYX_NOTE_AIR_5C,

--- a/app/src/main/java/org/koreader/launcher/device/LightsFactory.kt
+++ b/app/src/main/java/org/koreader/launcher/device/LightsFactory.kt
@@ -19,6 +19,7 @@ object LightsFactory {
                 DeviceInfo.Id.ONYX_GO6,
                 DeviceInfo.Id.ONYX_GO7,
                 DeviceInfo.Id.ONYX_GO7GEN2,
+                DeviceInfo.Id.ONYX_NOTE_AIR_3,
                 DeviceInfo.Id.ONYX_NOTE_AIR_3C,
                 DeviceInfo.Id.ONYX_NOTE_AIR_4C,
                 DeviceInfo.Id.ONYX_NOTE_AIR_5C,

